Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocDateTimeValue.css

```diff
@@ -22,72 +21,24 @@ span.bocDateValue
   min-width: 8em;
 }
 
-span.bocDateTimeValue.readOnly,
-span.bocDateValue.readOnly
-{
-  height: inherit;
-  /* Ensures that the outline is visible in Firefox. */
-  overflow: visible;
-}
-
-span.bocDateTimeValue.disabled,
-span.bocDateValue.disabled
-{
-}
-
-.mozilla span.bocDateInputWrapper
-{
-  right: 2px;
-}
-
 span.bocDateInputWrapper
 {
-  position: absolute;
-  left: 0;
-  min-width: 4em;
-  width: auto;
-}
-
-span.bocTimeInputWrapper
-{
-  position: absolute;
-  right: 0;
-}
-
-span.bocDateTimeValue span.bocDate24HoursWithSeconds,
-span.bocDateTimeValue span.bocDate12HoursWithSeconds
-{
-  right: 9.0em;
-}
-
-.mozilla span.bocDateTimeValue span.bocDate24HoursWithSeconds,
-.mozilla span.bocDateTimeValue span.bocDate12HoursWithSeconds
-{
-  right: calc(9.0em + 2px);
-}
-
-span.bocDateTimeValue span.bocDate24Hours,
-span.bocDateTimeValue span.bocDate12Hours
-{
-  right: 7.4em;
-}
-
-.mozilla span.bocDateTimeValue span.bocDate24Hours,
-.mozilla span.bocDateTimeValue span.bocDate12Hours
-{
-  right: calc(7.4em + 4px);
+  width: 100%;
```

Comment:
No, this cannot be removed, the property ensures that the entire space is always used by the element. It guarantees that the row is always the same size regardless of the number of elements in it:
![Screenshot 2021-08-19 075140](https://user-images.githubusercontent.com/48289242/130015465-c8b1219c-dbc8-44ef-986a-ef5fdacc31a0.png)
Without it, the row with two elements would be smaller than the one with three elements:
![Screenshot 2021-08-19 075236](https://user-images.githubusercontent.com/48289242/130015410-f4534302-91b2-4a25-a2e9-46dd1ca8a398.png)

Reply:
nvm, this behaviour no longer occurs if the `bocDateInputWrapper input` element is a border-box.

Reply:
Arggh..., `width: 100%` is needed because in Chrome/Edge the `input` element uses the default width, which is smaller than the available width. In Firfox it automatically uses the max available space.